pkg/labels/parse_test.go

```diff
@@ -245,6 +330,28 @@ func TestMatchers(t *testing.T) {
 			input: `"foo="bar""`,
 			err:   `bad matcher format: "foo="bar""`,
 		},
+		{
+			input: `{{foo=`,
+			err:   `bad matcher format: {foo=`,
+		},
+		{
+			// This looks like a bug which should be fixed
```

Comment:
```suggestion
```

Reply:
I suggest we don't leave these tasks as comments in the code and instead either: a) open individual PRs to fix them or b) keep them in your task list of things to do so that we can discuss them individually.